client/material/tabs.tsx

```diff
@@ -174,5 +177,52 @@ export function Tabs<T>({ children, activeTab, onChange, className }: TabsProps<
     return tabElems
   }, [activeTab, children, onChange])
 
+  useKeyListener({
+    onKeyDown: (event: KeyboardEvent) => {
+      if ((event.keyCode === PAGEUP || event.keyCode === PAGEDOWN) && event.ctrlKey === true) {
+        const enabledChildren: React.ReactElement[] = []
+        React.Children.forEach(children, (child, i) => {
+          if (child && !(child.props as TabItemProps<T>).disabled) {
+            enabledChildren.push(child)
+          }
+        })
+
+        if (enabledChildren.length < 2) {
+          return false
+        }
+
+        let previousOrNextTab: T | undefined
+        React.Children.forEach(enabledChildren, (child, i) => {
+          let previousOrNextChild
+          if (event.keyCode === PAGEUP) {
+            previousOrNextChild =
+              i === enabledChildren.length - 1 ? enabledChildren[0] : enabledChildren[i + 1]
```

Comment:
The logic on this line seems backwards to me (they hit page up, but you're going to the next tab?). You can also use modulus to do this without a ternary:

previous: `(i - 1 + enabledChildren.length) % enabledChildren.length`
next: `(i + 1) % enabledChildren.length`

Reply:
I did think (or rather found a stackoverflow post :p) of using the modulus operator, but to be perfectly honest I just find that a lot less readable/understandable.